### DISCUSSION TITLE
github pages hosting issues

### DISCUSSION THREAD

**Discussion by Tevin-wd** [community/community#40489] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am trying to host a small site for school to be reviewed by my peers and when I follow the steps and deploy the site with the correct repository name I get a 404 error saying that I do not have an index.html file in my root folder.

In troubleshooting, it says to have the index.html file at the top. I tried that and it didn't work. I then thought maybe it was because I have additional folders in there so I tried with another test repo with a very basic HTML file only in it and I still get the same error. 

not sure what  I am doing honestly but any help would be great as I need a link to provide to my course instructor and not really sure how else to provide a link with out getting the wite hosted. I really only need it hosted for like a week or two. 

Any help would be great!

screenshot showing I have deployed
![2022-11-30 16_26_15-Pages](https://user-images.githubusercontent.com/52752643/204929816-9d0bbbc7-6250-41d1-b81c-dd45c34210ee.png)

screenshot showing my root folder with index.html file
![2022-11-30 16_29_50-Taniti](https://user-images.githubusercontent.com/52752643/204930216-f9162094-601e-4564-a208-c376f339bf0e.png)

**Comment by radar-blip** [comment, marked as answer]
You have all your files in folders (which is why its not working) you need to at least have your index.html in the main section of your repository.
Then you can adjust links accordingly.

**Reply by Tevin-wd** [reply]
thank you this helped

**Comment by ihnatenkoo** [comment]
Hello, rename your repository to `tevin-wd.github.io`

**Comment by jfullstackdev** [comment]
you need to fix it by correcting your name as the name also of the page

![image](https://user-images.githubusercontent.com/47092464/205429270-55052132-64cf-4f51-8ffc-9b6dd2a4485f.png)

check this out too 

https://github.com/jdevstatic/github-pages-tutorial
